frictionless/field.py

```diff
@@ -58,6 +58,7 @@ def __init__(
         float_number=None,
         decimal_char=None,
         group_char=None,
+        example=settings.DEFAULT_EXAMPLE,
```

Comment:
Let's remove DEFAULT_EXAMPLE as, currently, the codebase style is not to use defaults like this. So it should be not present in the metadata dict if not provided and present if provided